client/my-sites/backup/backup-contents-page/file-browser/use-file-browser-state.ts

```diff
@@ -412,7 +412,6 @@ export function useFileBrowserState(): FileBrowserStateActions {
 		return checkList;
 	}, [ state.rootNode, addChildrenToList ] );
 
-	// Public interface matching Redux selectors
```

Comment:
I’d prefer not to have any more Redux mentions inside the FileBrowser component :)